frontend/src/components/pages/HooksDemo/emojis.ts

```diff
@@ -0,0 +1,67 @@
+export type EmojiInfo = {
+  name: string;
+  emoji: string;
+  description: string;
+  uses?: string[];
+};
+
+export const EMOJIS: EmojiInfo[] = [
+  {
+    name: "Smiley face",
+    emoji: "🙂",
+    description: "The simplest smiley face emoji :).",
+    uses: [
+      "possible passive aggression",
+      "said you're fine but actually dead inside",
+      "customer service",
+    ],
+  },
+  {
+    name: "Scream",
+    emoji: "😱",
+    description: "An emoji screaming.",
+    uses: ["AAAAAAAAAAAAA", "broke main", "exams"],
+  },
+  {
+    name: "Skull",
+    emoji: "💀",
+    description: "Just a skull.",
+    uses: ["aaaaaaa", "laughter(?)"],
+  },
+  {
+    name: "Smiling with tear",
+    emoji: "🥲",
```

Comment:
This one doesn't render on Windows unfortunately since it's fairly new. Not a bit deal but would be nice to use one that's compatible with all platforms, you can pick whichever one you want